docs/pages/guides/using-sentry.mdx

```diff
@@ -212,37 +213,64 @@ With `expo-updates`, release builds of both iOS and Android apps will create and
 
 - Your `release` will automatically be set to Sentry's expected value- `${bundleIdentifier}@${version}+${buildNumber}` (iOS) or `${androidPackage}@${version}+${versionCode}` (Android).
 - Your `dist` will automatically be set to Sentry's expected value: `${buildNumber}` (iOS) or `${versionCode}` (Android).
-- The configuration for build time sourcemaps comes from the `ios/sentry.properties` and `android/sentry.properties` files. For more information, refer to [Sentry's documentation](https://docs.sentry.io/clients/java/config/#configuration-via-properties-file). Manual configuration is only required for bare projects, the [sentry-expo config plugin handles it otherwise](#add-the-config-plugin).
-- Configuration for `expo publish` and `npx expo export` for projects is done via `app.json`, whether using bare workflow or not.
+- The configuration for build time sourcemaps comes from the **ios/sentry.properties** and **android/sentry.properties** files. For more information, refer to [Sentry's documentation](https://docs.sentry.io/clients/java/config/#configuration-via-properties-file). Manual configuration is only required for bare projects, the [sentry-expo config plugin handles it otherwise](#add-the-config-plugin).
+- Configuration for `expo publish` and `npx expo export` for projects is done via **app.json**, whether using bare workflow or not.
```

Comment:
Not sure about this but do we still need reference global Expo CLI commands? And if yes, I think it will be suffixed by `expo-cli`?

Reply:
🤔 You are right we don't specify using `expo-cli` as a command and we don't have a standard as of now. I think this can be marked as resolved and we can circle back later to update commands if required in future.

Reply:
Sounds good. Thanks for your sharp eye though!